django_prometheus/middleware.py

```diff
@@ -2,6 +2,26 @@
 from django_prometheus.utils import Time, TimeSince, PowersOf
 import django
 
+DEFAULT_LATENCY_BUCKETS = (
+    0.01,
+    0.025,
+    0.05,
+    0.075,
+    0.1,
+    0.25,
+    0.5,
+    0.75,
+    1.0,
+    2.5,
+    5.0,
+    7.5,
+    10.0,
+    25.0,
+    50.0,
```

Comment:
from a performance perspective, do we need these buckets?
from the man himself, 
> You can define custom buckets for latency, adding more buckets decreases performance but increases accuracy

https://github.com/korfuri/django-prometheus#configuration

Reply:
if we only add the bucket 75.0, we would be able to set an alert on let's say 60.0, any any latency above 75.0 we shouldn't even care